dart/lib/src/hub.dart

```diff
@@ -309,43 +314,95 @@ class Hub {
     }
   }
 
-  Map<String, String> traceHeaders() {
-    return {'sentry-trace': ''};
-  }
+  // Map<String, String> traceHeaders() {
+  //   return {'sentry-trace': ''};
+  // }
+
+  ISentrySpan startTransaction(
+    String name,
+    String operation, {
+    String? description,
+    bool? bindToScope,
+  }) =>
+      startTransactionWithContext(
+        SentryTransactionContext(
+          name,
+          operation,
+          description: description,
+        ),
+        bindToScope: bindToScope,
+      );
 
-  void startTransaction() {}
+  ISentrySpan startTransactionWithContext(
```

Comment:
to discuss: since Dart does not have method overloadings, either we use different method names or fit every possible parameter as optional, but this could cause confusion since you may pass `transactionContext` that takes a `name` and `name` but with different values

Reply:
let's keep like this, similar to iOS